packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html

```diff
@@ -1005,6 +1005,15 @@
                             </tr>
                         </tfoot>
                     </table>
+                    <div>
+                        <p>
+                            {{'telecom_pack_migration_offers_debit_mention_1' |
+                            translate}}
+                            <br />
+                            {{'telecom_pack_migration_offers_debit_mention_2' |
+                            translate}}
+                        </p>
+                    </div>
```

Comment:
```suggestion
                        <p class="mb-0" data-translate="telecom_pack_migration_offers_debit_mention_1"></p>
                        <p class="mb=0" data-translate="telecom_pack_migration_offers_debit_mention_2"></p>
```

Reply:
Sorry but I want to understand what does your method bring more than mine? Better performance, better accessibility,..? I don't understand why you insist on this point.

Reply:
Hello @Steffy29, `div` element is a container where `p` is for displaying text.